capstone/src/main/java/com/google/step/data/RestaurantPageViews.java

```diff
@@ -0,0 +1,51 @@
+package com.google.step.data;
+
+import java.util.List;
+
+/**
+ * Object that contains all the weekly page views of a specific restaurant.
+ */
+public class RestaurantPageViews {
+
+    private final String name;
+    private final List<WeeklyPageView> pageViews;
+
+    public RestaurantPageViews(String name, List<WeeklyPageView> pageViews) {
+        this.name = name;
+        this.pageViews = pageViews;
+    }
+
+    /**
+     * Gets the name of the instance
+     * @return String representing the name of the restaurant.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Gets the list of weeklyPageViews of the restaurant
+     * @return list of weekly page views
+     */
+    public List<WeeklyPageView> getPageViews() {
+        return pageViews;
+    }
+
+    /**
+     * Equals method for comparisons
+     * @param other object to compare to
+     */
+    @Override
+    public boolean equals(Object other) {
```

Comment:
Also implement hashCode. It's considered a requirement that if two objects are equal, then their hashCode MUST also be equal. By default, hashCode is just Object.hashCode, which uses the object reference, which violates this contract.

Reply:
Done!